src/pages/ResearcherProfile.js

```diff
@@ -525,531 +356,448 @@ export const ResearcherProfile = hh(class ResearcherProfile extends Component {
+                      'As a "Signing Official", your institution cannot be changed here. Please submit a support request via the "Contact Us" form to have it changed.' :
+                      'If your preferred institution cannot be found, please submit a support request via the "Contact Us" form to have it added.',
+                    'data-for': 'tip_profileState',
+                  })
                 ]),
+                generateInstitutionSelectionDisplay()
+              ]),
 
-                div({ className: 'col-xs-12 no-padding' }, [
-                  div({ className: 'row fsi-row-lg-level fsi-row-md-level no-margin' }, [
-                    div({ className: 'col-xs-6 rp-group' }, [
-                      label({ id: 'lbl_profileZip', className: 'control-label' }, ['Zip/Postal Code*']),
-                      input({
-                        id: 'profileZip',
-                        name: 'zipcode',
-                        type: 'text',
-                        className: (this.state.invalidFields.zipcode && showValidationMessages) ?
-                          'form-control required-field-error' :
-                          'form-control',
-                        onChange: this.handleChange,
-                        value: isNil(this.state.profile.zipcode) ? '' : this.state.profile.zipcode,
-                        required: true
-                      }),
-                      span({
-                        className: 'cancel-color required-field-error-span',
-                        isRendered: this.state.invalidFields.zipcode && showValidationMessages
-                      }, ['Zip/Postal Code is required'])
-                    ]),
-
-                    div({ className: 'col-xs-6 rp-group' }, [
-                      label({ id: 'lbl_profileCountry', className: 'control-label' }, ['Country*']),
-                      select({
-                        id: 'profileCountry',
-                        name: 'country',
-                        onChange: this.handleChange,
-                        value: this.state.profile.country,
-                        className: (this.state.invalidFields.country && showValidationMessages) ?
-                          'form-control required-field-error' :
-                          'form-control',
-                        required: true
-                      }, countryNames ),
-                      span({
-                        className: 'cancel-color required-field-error-span',
-                        isRendered: this.state.invalidFields.country && showValidationMessages
-                      }, ['Country is required'])
-                    ])
+              div({ className: 'col-xs-12 no-padding' }, [
+                div({ className: 'row fsi-row-lg-level fsi-row-md-level no-margin' }, [
+                  div({ className: 'col-xs-6' }, [
+                    label({ id: 'lbl_profileDepartment', className: 'control-label' }, ['Department']),
```

Comment:
It looks like this went from required to non-required with the last update. Can we revert these changes as part of this PR?

Reply:
This was part of the discussion in today's standup. Department was changed to non-required on other parts of the site, and they asked me to change it on this page as well.

Reply:
Gotcha, thanks, wasn't aware of that 👍🏽